website/i18n/cn/docusaurus-plugin-content-pages/syncs.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 2
+title: "Community Syncs"
+toc: true
+last_modified_at: 2020-09-01T15:59:57-04:00
+---
+
+# 社区进展同步
```

Comment:
@minihippo Can you share a screenshot image of the rendered page?

Reply:
It looks like 
![image](https://user-images.githubusercontent.com/17903481/176849577-ab782e1c-ca89-4a47-85ec-29377b666655.png)